blazor/rich-text-editor/webassembly-performance.md

```diff
@@ -0,0 +1,132 @@
+* Now, RichTextEditor component will not be a part of the rendering which happens as result of the click event and **currentCount** alone will get updated.
+
+```cshtml
+@using Syncfusion.Blazor.RichTextEditor
+
+<p>Current count: @currentCount</p>
+
+<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
+
+<SfRichTextEditor @ref="rteObj"> 
+    <RichTextEditorToolbarSettings Items="@Tools" Type="ToolbarType.Expand" />
+    <p>The Rich Text Editor component is WYSIWYG ('what you see is what you get') editor that provides the best user experience to create and update the content. Users can format their content using standard toolbar commands.</p> 
+</SfRichTextEditor> 
+@code {
+    SfRichTextEditor rteObj;
+    private int currentCount = 0;
+    private void IncrementCount()
+    {
+        rteObj.PreventRender();
+        currentCount++;
+    };
+    private List<ToolbarItemModel> Tools = new List<ToolbarItemModel>() 
+    { 
+        new ToolbarItemModel() { Command = ToolbarCommand.Bold }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Italic }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Underline }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.StrikeThrough }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.FontColor }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.BackgroundColor }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Formats }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Alignments }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.OrderedList }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.UnorderedList }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Outdent }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Indent }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator },
+        new ToolbarItemModel() { Command = ToolbarCommand.CreateLink }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Image }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.CreateTable }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Undo }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Redo } 
+    };
+}
+```
+
+N> * **PreventRender** method accepts boolean argument that accepts true or false to disable or enable rendering respectively.
+<br/> * **PreventRender** method can be used only after RichTextEditor component completed initial rendering. Calling this method during initial rendering will not have any effect.
```

Comment:
<br/> * **PreventRender** method can be used only after the RichTextEditor component completes initial rendering. Calling this method during initial rendering will not have any effect.

Reply:
addressed